lib/brains/management/commands/braintree_config.py

```diff
@@ -100,4 +107,29 @@ def handle(self, *args, **options):
                 if product.recurrence:
                     # If there's recurrence, we need to check
                     # that it exists in Braintree and is set up ok.
-                    product_exists(plans, product.id, product.amount)
+                    try:
+                        product_exists(plans, product.id, product.amount)
+                    except BraintreePlanDoesNotExist:
+                        raise CommandError(textwrap.dedent('''\
+                            Missing product: {product.id}
+
+                            Currently it's not possible to automate this.
+
+                            1. Log into https://sandbox.braintreegateway.com/
+                               (or the production dashboard)
+                            2. Go to Recurring Billing > Plans from the sidebar
+                            3. Enter the following data:
+
+                                Plan ID: {product.id}
+                                Plan Name: {product.description}
+                                Price: {price}
+                                Billing Cycle Every: 1 month
+
+                            4. re-run this script
+                            5. MFBT
```

Comment:
This is cute, but might look obscure to someone not indoctrinated into old Mozilla speak. I say add a link or make it a bit more generic. I wouldn't want someone to get confused when they're really at the end!

The :beer: emoji is an excellent alternative :wink: